crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -827,6 +1083,25 @@ fn is_generator_function_annotated_as_returning_none(
         .is_some_and(GeneratorOrIteratorArguments::indicates_none_returned)
 }
 
+fn parameters_from_signature<'a>(
+    docstring: &'a Docstring,
+    semantic: &'a SemanticModel,
+    dummy_variable_rgx: &'a Regex,
+) -> Vec<&'a str> {
+    let mut parameters = Vec::new();
+    let Some(function) = docstring.definition.as_function_def() else {
+        return parameters;
+    };
+    for param in function.parameters.iter().skip(usize::from(
+        docstring.definition.is_method() && !is_staticmethod(&function.decorator_list, semantic),
+    )) {
```

Comment:
nit: can we add a small comment stating that this logic is to skip the `self` argument if it's present?

Also, do we need to do the same for `cls` variable if it's a class method?

Reply:
Added comment.

The logic already works for classmethods.